### ISSUE TITLE
missing styles?

### ISSUE THREAD

**Issue by mstruensee** [AllenFang/react-bootstrap-table#1619]
I am including react-bootstrap-table-all.min.css in my scss file, but a lot of styles / option tags are not working (condensed for example) ...

My application is using bootstrap 4, is this causing these to not work? or am I missing another css that the live examples use?

Thanks!

Edit: using  version="4" in <BootstrapTable> also causing my filter boxes to not align.  Other issues still exist.

Edit: using version="4" is still adding the "table-condensed" class, which I think is the problem.  This is mentioned in the migration documentation:
 - Renamed `.table-condensed` to `.table-sm` for consistency.

**Comment by AllenFang** [comment]
if you upgrade the css before, please remember to replace newest css.

For the bootstrap 4, add version="4" is correctly, but please summary the issue you have, 

Now I can only reproduce the condensed, but I dont know what's issue for filter

Also, I'll not handle the issues which is not clear, like(Styles is not work.... etc), but what style.

**Comment by romgrk** [comment]
@AllenFang will you fix the `.table-condensed -> .table-sm` or should I open a new issue specifically for that? (if so, maybe rename the title of this issue)

A workaround for the `.table-sm` issue is to apply the class on a wrapper, because the corresponding rule is `.table-sm th { ... }`, thus it gets applied even if the class is not on the table element. E.g.
```jsx
const render = () =>
  <div className='table-sm'>
    <BootrastrapTable>
      /* ... */
    </BootrastrapTable>
  </div>
```

**Comment by AllenFang** [comment]
@romgrk sorry for lately reply, `table-sm` not fix yet, I'll fix it soon, sorry.

**Comment by AllenFang** [comment]
`table-condensed` issue was fixed on `v4.1.0`, let me know if any problem, thanks
